src/v1/components/surfaces/FloatingPanel/DraggableFloatingPanel.tsx

```diff
@@ -79,13 +118,18 @@ const DraggableFloatingPanel: React.FC<DraggableFloatingPanelProps> = ({
           {icon}
           <Text sx={{ paddingRight: 4 }}>
             {title}
-            {count && (
+            {count && count > 0 ? (
```

Comment:
Non-blocking question: Looks like this change wasnt strictly needed. Is this just personal pref or is something deeper goin on?

Reply:
I've updated the code, hopefully it makes more sense now